eng/Version.Details.xml

```diff
@@ -12,6 +12,10 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>63715256cc94963d608ef3d683d2529332c5140c</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.TemplateEngine.Tasks" Version="7.0.100-alpha.1.21601.1">
```

Comment:
Do we need to add a Maestro subscription for this? Can it be a dependency of dotnet/installer, so when we get updates for dotnet/installer this number follows along?

See an example here on how ILLink updates along with dotnet/installer: https://github.com/xamarin/xamarin-android/pull/6670

Reply:
Adding dependency is possible, but updates will be coming quite frequently - in templating all the packages are produced on any change (even unrelated to package).

Alternative is to add the task via Arcade SDK,  I see the repo has already reference to it https://github.com/xamarin/xamarin-android/blob/71e6ecd67b08394482317c94db9ff35dc051ffbc/global.json#L7 and dependency https://github.com/xamarin/xamarin-android/blob/71e6ecd67b08394482317c94db9ff35dc051ffbc/eng/Version.Details.xml#L21-L24   

But this version is very old, we need to update it. Also template localization task is only available from arcade SDK 7, if you need to stay on 6 - we might need to backport it.

Reply:
yeah need to know for us too.

Reply:
@pjcollins should we setup a weekly subscription for dotnet/arcade?

We might also need to configure a few things so the version number is used throughout our build.